apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/observe/api_collections/RunTest.jsx

```diff
@@ -100,7 +100,14 @@ function RunTest({ endpoints, filtered, apiCollectionId, disabled, runTestFromOu
         })
         testRoles.unshift({"label": "No test role selected", "value": ""})
         setTestRolesArr(testRoles)
-        const businessLogicSubcategories = allSubCategoriesResponse.subCategories
+
+        const customOnly =  allSubCategoriesResponse.subCategories.filter((x) => x.author !== 'AKTO')
+        const aktoOnly = allSubCategoriesResponse.subCategories.filter((x) => x.author === 'AKTO')
+        aktoOnly.sort((a,b)=>{
+            return a.updatedTs - b.updatedTs
+        })
+
+        const businessLogicSubcategories = func.checkLocal() ? [...customOnly, ...aktoOnly.slice(0, aktoOnly.length - 200)] : allSubCategoriesResponse.subCategories
```

Comment:
Oh. Lets not do this. This will remove any 200 on reboot. We should be specific in removing 200 tests. Not just any random 200.